services/cloudaccounts/aws/aws.go

```diff
@@ -44,6 +51,21 @@ type CloudAccountResponse struct {
 	LambdaScanner          bool                    `json:"lambdaScanner"`
 }
 
+type ProtectIAMEntitiesResponse struct {
+	RolesArn []Arn `json:"rolesArns"`
+	UsersArn []Arn `json:"usersArns"`
+}
+
+type Arn struct {
```

Comment:
It's confusing to use this struct name `Arn` especially when you have a field called `Arn` inside. Why calling it Arn? Arn is an AWS string identifier. Lets not use it for different types